lib/ReactViews/Generic/Editor.jsx

```diff
@@ -9,8 +9,8 @@ import "tinymce/icons/default";
 import "tinymce/themes/silver";
 import "tinymce/models/dom";
 /* Import a skin (can be a custom skin instead of the default) */
-// import "!!style-loader!css-loader?sourceMap!tinymce/skins/ui/oxide/skin.min.css";
-import "!!style-loader!css-loader?sourceMap!./editor.skin.min.css"; // Custom borderless skin
+// import "!!style-loader!css-loader!tinymce/skins/ui/oxide/skin.min.css";
+import "!!style-loader!css-loader!./editor.skin.min.css"; // Custom borderless skin
```

Comment:
The style-loader seems to have been removed from the webpack config, do these imports work as intended despite that?